### ISSUE TITLE
Design Intent - Accessibility Feedback - My Education Benefits, 22-5490 Form (Fry/DEA) Updates

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#95297]
## Next Steps for the VFS team

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- Sorry for the delayed feedback! Please reach out via Slack if you have any questions.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Must:** Echoing Allison's feedback for design, make sure you update all of the old components to their current iterations. Some of the newer versions include some accessibility improvements and it would be unfortunate if you missed out on them, even in a fast and small update.
- [ ] **Must:** There are a couple of instances where a question has explanatory helper text, eg. in step 2 "Which education benefit..." question, with the info component placed between the question and the radio buttons. There's a risk that screen reader users may miss out on any critical information included in that kind of helper text.
   
   When a screen reader user navigates through a form flow, they're typically trying to get through it as efficiently as possible --- like anyone else. That often means that they're tabbing through each interactive element on the page (buttons, links, inputs) rather than having their screen reader announce the whole page content. When tabbing through, the only text they here is text that's programmatically associated with the input. Any information that's not programmatically associated is likely to be missed.
   
   If it's text that's not critical to completing the task, then that's not a huge issue. But if it's information that a user should really know before they make a selection (eg. key definitions, eligibility criteria, etc) then missing that content is a big deal.
   
   We have a few possible ways to address this and associate helper text with an input. Totally up to your team for what the right solution is in a given situation.
      - Use the `hint` prop in one of the VADS components. The `hint` value populates on-screen helper text that's programmatically associated with the input.
      - Use the `header-aria-describedby` prop in VADS components that support it. The `header-aria-describedby` value populates visually hidden text that's programmatically associated with the input. Where appropriate, it may be duplicative of visible text elsewhere on the page.
      - Wrap the input(s) with a `fieldset` and place all critical helper text inside the `legend` element. Note that you can have multiple elements inside a `legend` and you can style them differently. When tabbing to the first input inside the `fieldset`, everything in the `legend` is announced. (However, text inside the `fieldset` but not in the `legend` does **not** get announced.) 

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [ ] Assign this ticket to the VFS team member that opened the Slack request
- [ ] Add the VFS team product label
- [ ] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [ ] Add the Collaboration Cycle initiative milestone
